### ISSUE TITLE
show_tabs template broken?

### ISSUE THREAD

**Issue by Mariusthvdb** [maykar/custom-header#94]
using this up to 1.1.7:

```
show_tabs: >-
  {% if not is_state('input_select.mode','Developer') %}0 to 3,5 to 22,27,29
  {% else %}0 to 29
  {% endif %}
```
now stops again  after tab 22.
fwiw, test_template results in:
<img width="643" alt="Schermafbeelding 2020-01-10 om 12 40 05" src="https://user-images.githubusercontent.com/33354141/72150710-75abc300-33a6-11ea-9dab-e216f459cc0a.png">

so even using >- there's a trailing white space. Not sure if this causes it though, because it was exactly the same on 1.1.7

hassio 103.5
same results using 
```
show_tabs: |
  {% if not is_state('input_select.mode','Developer') %}0 to 3,5 to 22,27,29
  {% else %}0 to 29
  {% endif %}
```
as does, for the sake of it:

```
show_tabs: "{% if not is_state('input_select.mode','Developer') %}0 to 3,5 to 22,27,29{% else %}0 to 29{% endif %}"
```

this does show:
```
Custom Header test returned: "0 to 3,5 to 22,27,29"
```
for the test_template though, which shows it should be another issue?

**Comment by Mariusthvdb** [comment]
and fixed with 1.2.0!
thanks for the immediate fix!

**Comment by maykar** [comment]
Thank you for finding it!
